st2auth_pam_backend/__init__.py

```diff
@@ -21,4 +21,4 @@
     'PAMAuthenticationBackend'
 ]
 
-__version__ = '0.2.0'
+__version__ = '0.3.0'
```

Comment:
Shouldn't we also start tagging these with the corresponding ST2 release version, they are gonna be released together anyways.

Reply:
Yeah. I bumped it as is, but versioning with st2 would be best. I think we can change that once we combine repos.